Tests/Shared.Specs/MethodInfoAssertionSpecs.cs

```diff
@@ -125,6 +126,49 @@ public void When_asserting_a_method_is_decorated_with_attribute_and_it_is_it_suc
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_asserting_a_method_is_decorated_with_MethodImpl_attribute_and_it_is_it_succeeds()
```

Comment:
If I know @dennisdoomen right, he prefers leaving out the `_asserting_` part of the test name as it doesn't add value.

Reply:
Sure, I just copied from previous tests, assuming it's the standard around.

Reply:
Yeah, we do have _a lot_ of superfluous  `_asserting_` is test names.
It is on my todo, but quite far down.